examples/tutorials/Advanced/Federated LSTM-Predicting Next Word.ipynb

```diff
@@ -0,0 +1,520 @@
+    "\n",
+    "Hrishikesh Kamath - GitHub: @<a href=\"http://github.com/kamathhrishi\">kamathhrishi</a>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "#Import required libraries\n",
+    "\n",
+    "import torch\n",
+    "import torch.nn as nn\n",
+    "import torch.optim as optim\n",
+    "import numpy as np\n",
+    "from torch.utils.data import Dataset"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "#Dataset consisting of a small Shakespere Passage\n",
+    "\n",
+    "test_sentence = \"\"\"When forty winters shall besiege thy brow,\n",
+    "And dig deep trenches in thy beauty's field,\n",
+    "Thy youth's proud livery so gazed on now,\n",
+    "Will be a totter'd weed of small worth held:\n",
+    "Then being asked, where all thy beauty lies,\n",
+    "Where all the treasure of thy lusty days;\n",
+    "To say, within thine own deep sunken eyes,\n",
+    "Were an all-eating shame, and thriftless praise.\n",
+    "How much more praise deserv'd thy beauty's use,\n",
+    "If thou couldst answer 'This fair child of mine\n",
+    "Shall sum my count, and make my old excuse,'\n",
+    "Proving his beauty by succession thine!\n",
+    "This were to be new made when thou art old,\n",
+    "And see thy blood warm when thou feel'st it cold.\"\"\".split()"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import syft as sy\n",
+    "\n",
+    "hook = sy.TorchHook(torch)  # <-- NEW: hook PyTorch ie add extra functionalities to support Federated Learning\n",
+    "bob = sy.VirtualWorker(hook, id=\"bob\")  # <-- NEW: define remote worker bob\n",
+    "alice = sy.VirtualWorker(hook, id=\"alice\")  # <-- NEW: and alice"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "<h1>Federated Dataset</h1>"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {
+    "collapsed": true
+   },
+   "outputs": [],
+   "source": [
+    "class TextDataset(Dataset):\n",
+    "\n",
+    "    def __init__(self,text,transform=None):\n",
+    "        \n",
+    "        \"\"\"arguments:\n",
+    "        \n",
+    "             text (List of Strings): Text corpus \n",
+    "             transform: List of transforms to be performed on the input data\n",
+    "             \n",
+    "        \"\"\"\n",
+    "\n",
+    "        self.text = text\n",
+    "        self.data=[]\n",
```

Comment:
always add spaces around equal sign for attribution: self.data=[] -> self.data = []